### PR TITLE
:rocket: Add GitHub Release.

### PR DESCRIPTION
**Pull request by ldez**
Create a tar with the `dist` content and publish in GH release.

### CODE REVIEW

**Comment by mojavelinux** [comment]
Nice!

However, I think we should create a zip instead of (or in addition to) a tar. As much as I hate to say it (and I do), zip is more accessible for the cross-platform web community than tar.

**Comment by ldez** [comment]
See https://github.com/ldez/ajsdeploy/releases

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
+
 sudo: false
 
 before_deploy:
-- npm run dist
+- >
+  if ! [ "$BEFORE_DEPLOY_RUN" ]; then
+    export BEFORE_DEPLOY_RUN=1;
+    npm run dist;
+    mkdir bin
```
Comment (Mogztter) [inline .travis.yml]: Should we run this command in `npm run dist` ?
The main advantage is that we can reproduce the task on a local environment. Wdyt ?

I can take care of this one, in another pull request.

**Comment by Mogztter** [comment]
LGTM

**Comment by ldez** [comment]
@Mogztter you need to add `GITHUB_TOKEN` in Travis GUI 😉

**Comment by Mogztter** [comment]
@ldez Done!

File: .travis.yml (hunk 1 above)
Comment (ldez) [inline .travis.yml]: 1. it's possible.
2. I don't known if we have real need to reproduce this specific GH step in local.
3. If you want, you commit in my PR

**Pull request merged**